pkg/graph/schema.graphql

```diff
@@ -1122,6 +1123,43 @@ type SystemIntakeActionActor {
   email: String!
 }
 
+"""
+GRB Reviewers for a system intake request
+"""
+type SystemIntakeGRBReviewer {
+  id: UUID!
+  userID: UUID!
```

Comment:
Per our convo -- let's remove this field in favor of `userAccount.id`

Reply:
Removed in [b0c8dda](https://github.com/CMSgov/easi-app/pull/2588/commits/b0c8dda76f539c0c4c09bd89ed5c704790f5eab8)